src/main/java/net/sf/mzmine/modules/masslistmethods/chromatogrambuilder/ChromatogramBuilderTask.java

```diff
@@ -171,6 +172,9 @@ public void run() {
 	// Add new peaklist to the project
 	project.addPeakList(newPeakList);
 
+        // Add quality parameters to peaks
+        new QualityParameters(newPeakList);

```

Comment:
what is the meaning of this line? 
it looks a bit strange to just make an instance of a class and not assign the instance to any variable.